src/background/redirects.js

```diff
@@ -29,4 +29,16 @@ function redirectFromGoogle(requestDetails) {
     }
 }
 
-export { redirectFromFandom, redirectFromGoogle }
+function redirectFromDdg(requestDetails) {
+    const url = new URL(requestDetails.url)
+    const searchQuery = url.searchParams
+        .get("q")
+        .replace(/ /g, "+")
+        .replace(/^poe\+/, "")
+
+    return {
+        redirectUrl: `https://www.duckduckgo.com/?q=site:poewiki.net+${searchQuery}`,
+    }
```

Comment:
(After merging/rebasing from master)

Note how the implementation of both this and `redirectFromGoogle` are identical - you should move these to a common `searchQueryFromRequest` function and remove the duplication entirely.

This will make future search engine support trivial to add.

Reply:
Also, please wait to merge/rebase from master, after #10 has been merged, to minimise any other conflicts. Thanks :)